aclgen.py

```diff
@@ -24,7 +24,9 @@
 from __future__ import unicode_literals
 
 __author__ = 'pmoody@google.com'
+__author__ = 'watson@google.com'
```

Comment:
author tags are no longer in style Tony ;) I have been considering removing all the existing ones from the code actually. I will share with you an internal document talking about this.